test/e2e/go.mod

```diff
@@ -34,15 +34,17 @@ replace (
 
 require (
 	github.com/google/go-cmp v0.5.5
-	github.com/onsi/ginkgo v1.14.0
-	github.com/onsi/gomega v1.10.1
+	github.com/onsi/ginkgo v1.16.4
+	github.com/onsi/gomega v1.14.0
+	github.com/ovn-org/ovn-kubernetes/go-controller v0.0.0-20220427175813-acd09730af95
```

Comment:
err, why are we re-vendoring ourselves? are you using your own egressqos files for the e2e's? I think we don't vendor or pull ourselves for the tests.... check the `e2e.go` I think we only use test frameworks?

Reply:
removed, moved to YAMLs instead of using the EgressQoS API